src/storage_broker/api/__init__.py

```diff
@@ -17,7 +23,14 @@ def get_archive_url():
     except ValueError:
         return error_message('invalid request_id'), 400
 
-    sample_response = {"request_id": request_id, "url": "https://example.com/", "timeout": "2022-04-14T19:58:51.598Z"}
+    url = aws.get_url(config.STAGE_BUCKET, request_id, config.API_URL_EXPIRY)
+    if not url:
+        return error_message('aws client error'), 500
+
+    timeout = datetime.utcnow() + timedelta(seconds=config.API_URL_EXPIRY)
+    timeout_str = str(timeout.replace(microsecond=0, tzinfo=timezone.utc).isoformat())
+
+    sample_response = {"request_id": request_id, "url": url, "timeout": timeout_str}
```

Comment:
This is a little nit-picky so feel free to ignore!  

I'm not sure if sample_response is the right name for this variable. Since the response is the legitimate response returned for this API endpoint and sample response gives the connotation of a "mocked" response.

Again!  Super tiny nit-pick, feel free to ignore!  😄

Reply:
Ha! Forgot to change that variable name! :laughing:

Reply:
Should be good now :+1:

Reply:
Awesome!  Looks great!